transfers/contact_transfer.py

```diff
@@ -99,21 +114,26 @@ def transfer_contacts(session):
             session.rollback()
 
 
-def _add_first_contact(session, row, thing):
+def _add_first_contact(session, row, thing, co_to_org_mapper):
     # TODO: extract role from OwnerComment
     # role = extract_owner_role(row.OwnerComment)
     role = "Owner"
     release_status = "private"
 
     name = _make_name(row.FirstName, row.LastName)
 
```

Comment:
suggestion: it's probably worth assigning a local `company` variable here, rather than keep reading from `row`